src/frontend/src/lib/components/license-agreement/LicenseAgreement.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+	import { Html } from '@dfinity/gix-components';
+	import { i18n } from '$lib/stores/i18n.store.js';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils.js';
+
+	let termsList: string[];
```

Comment:
`argreementList` if we rename everything?

Reply:
make sense